src/wp-admin/includes/image.php

```diff
@@ -482,6 +482,60 @@ function _wp_make_subsizes( $new_sizes, $file, $image_meta, $attachment_id ) {
 	return $image_meta;
 }
 
+/**
+ * Copy parent attachment properties to newly cropped image.
+ *
+ * @since 6.5.0
+ *
+ * @param string $cropped              Path to the cropped image file.
+ * @param int    $parent_attachment_id Parent file Attachment ID.
+ * @param string $context              Control calling the function.
```

Comment:
Let's make `$context` an optional argument with a default value of ''. While all the existing uses of this function require it, the function will have broader utility if it's optional.